lint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "inside $0"
+
+RET=0
+
+flake8 statsmodels --select=E901,E999,F822,F823,E111,E114
+if [ $? -ne "0" ]; then
```

Comment:
Why not just __exit $?__

Reply:
Because I anticipate needing to add more calls as we incrementally fix stuff up.  e.g.

```
flake8 statsmodels --select=F821 --exclude=LIST_OF_FILES_WE_INCREMENTALLY_FIX
if [ $? ...

flake8 statsmodels --select=??? --exclude=LIST_OF_FILES_WE_INCREMENTALLY_FIX
if [ $? ...

```

It's not pretty, but all-at-once fixes aren't really workable here.